clients/js/src/AdminClient.ts

```diff
@@ -74,10 +74,6 @@ export class AdminClient {
         ...this.api.options.headers,
         ...this.authProvider.authenticate(),
       };
-      this.api.options.headers = {
```

Comment:
These lines are exactly the same as the four above so should have no effect, I think.